tests/training/test_sample_training_response.py

```diff
@@ -62,7 +62,7 @@ def test_predict():
 
     #  Teddy Bear
     # assert response['predictions'][0]['label_id'] == '88'
-    assert response['predictions'][0]['label'] == 'toy'
+    assert (response['predictions'][0]['label'] == 'toy' or response['predictions'][0]['label'] == 'pen')
```

Comment:
```suggestion
    assert response['predictions'][0]['label'] in ('toy', 'pen')
```

Reply:
Why does this sometimes return different results?

Reply:
My guess is that there's some randomness in the training process and the result can vary. What I would do is to use a different image that produces more consistent results, but if it's too difficult I would at least add a comment explaining what's going on here.